Gems/ONNX/README.md

```diff
@@ -38,3 +38,139 @@ This is an experimental gem implementing [ONNX Runtime](https://onnxruntime.ai/)
 - [CUDA Toolkit](https://developer.nvidia.com/cuda-toolkit) v11.4 or greater.
 - [CUDNN library](https://developer.nvidia.com/cudnn) v8.2.4 or greater.
 - [zlib](https://zlib.net/) v1.2.3 or greater.
+
+## Additional developer docs
+
+### **What is ONNX?**
+- [Open Neural Network Exchange](https://onnx.ai/)
+- An open standard for machine learning models, defining an extensible computation graph model, operators and data types
+- Its key benefit is interoperability - the idea is that you can generate a model using different ML frameworks (such as Tensorflow, Keras, PyTorch), export it as an ONNX model, and in theory these models should behave the same
+- It has also recently become a member of the [Linux Foundation](https://www.linuxfoundation.org/)
```

Comment:
nit: It would be good to have periods/full-stops at the end of the sentences to be consistent with the above style

Reply:
Changed